WMass/python/plotter/w-helicity-13TeV/wmass_e/systsEnv.txt

```diff
@@ -32,7 +32,7 @@ CMS_We_elescale      :  W.*: .* : elescale         : templates
 # non-flat normalization uncertainty
 CMS_We_FRe_norm      : data_fakes  : .* : FRe_norm : templates
 # shape uncertainty (slope of FR with pt)
-CMS_We_FRe_pt        : data_fakes  : .* : FRe_pt   : templatesShapeOnly
+#CMS_We_FRe_pt        : data_fakes  : .* : FRe_pt   : templatesShapeOnly
```

Comment:
let's keep in mind that we need to introduce it back as a pt-dependent shape uncertainty (@cippy says that this is temporary removed because the FR is now parameterized as a parabola and taking +/- one sigma of the linear coeff doesn't make sense).